orchagent/neighorch.cpp

```diff
@@ -393,9 +393,18 @@ bool NeighOrch::addNeighbor(NeighborEntry neighborEntry, MacAddress macAddress)
         status = sai_neighbor_api->create_neighbor_entry(&neighbor_entry, 1, &neighbor_attr);
         if (status != SAI_STATUS_SUCCESS)
         {
-            SWSS_LOG_ERROR("Failed to create neighbor %s on %s, rv:%d",
+            if (status == SAI_STATUS_ITEM_ALREADY_EXISTS)
+            {
+                SWSS_LOG_ERROR("Entry exists: neighbor %s on %s, rv:%d",
                            macAddress.to_string().c_str(), alias.c_str(), status);
-            return false;
```

Comment:
function addNeighbor() didn't comment the meaning of false/true. This is an issue that I don't think you have to fix in this PR.

But maybe you could add a comment here to state why returning true when error code is ALREADY_EXISTS?

Reply:
Added a comment